HISTORY.md

```diff
@@ -22,13 +21,14 @@ http://visjs.org
 - FIX #3163: Prevent crashes from invalid id's in `Clustering.findNode()`
 - FIX #3106: Ensure start and end of stabilization progress events is sent
 - FIX #3015: Properly handle newline escape sequences in strings for DOT
-- FIX Refactoring of LayoutEngine
+- FIX: Refactoring of LayoutEngine (#3110)
```

Comment:
Yeah, it's different because it's not a fix for a particular issue.

Open a full view of the file and look at the entries of previous releases. They are of the form:

```
- FIX #<issue number> <description> (#<PR number>)
```
...where `#<issue number>` is the issue fixed by this PR. If there's no related issue, then `#<issue number>` is left out.

I'd really like to go the other way and use that same layout for the latest release.